pymc/model/fgraph.py

```diff
@@ -158,6 +160,12 @@ def fgraph_from_model(
             "Nested sub-models cannot be converted to fgraph. Convert the parent model instead"
         )
 
+    check_for_gp_vars = [
+        k for x in ["_rotated_", "_hsgp_coeffs_"] for k in model.named_vars.keys() if x in k
+    ]
+    if len(check_for_gp_vars) > 0:
+        warnings.warn("Unable to clone Gaussian Process Variables", UserWarning)
```

Comment:
This can be simplified

```suggestion
    if any(name in ("_rotated_", "_hsgp_coeffs_") for name in model.named_vars):
        warnings.warn("Unable to clone Gaussian Process Variables", UserWarning)
```

Reply:
Yeah, that looks better. I will make that change. Thank you!

Reply:
Oh, hey @ricardoV94. Since we moved that check to the `fgraph_from_model()` function maybe the warning should say something different instead of mentioning cloning? Is this function only used for cloning?

Reply:
Hey, one more thing. I wasn't able to get the above suggestion to work. I think that we need to check if the string subset ("_rotated_" & "_hsgp_coeffs_") are in the name of the variable. I was able to get it to work by making a little change:

```
if any(gp_name in var_name for gp_name in ["_rotated_", "_hsgp_coeffs_"] for var_name in model.named_vars):
        warnings.warn("Unable to clone Gaussian Process Variables", UserWarning)
```
Is that okay with you?

Reply:
Oh sorry of course. Since there are only two options I would do an `or` statement then: `if '_rotated_` in var_name or '_hsgp_coeffs_o in var_name for var_name in ...`

Reply:
Hey @ricardoV94, I couldn't figure out how to get it to work by only looping through model.named_vars one time like you have in your example. I had to do a loop in each one of the `or` cases like this: 

```
if any("_rotated_" in var_name for var_name in model.named_vars) or any("_hsgp_coeffs_" in var_name for var_name in model.named_vars):
        warnings.warn("Unable to clone Gaussian Process Variables", UserWarning)
```

Am I missing something?

Reply:
`if any(("_rotated_" in var_name or "_hsgp_coeffs_" in var_name) for var_name in model.named_vars)`

Reply:
Also let's give a more informative warning. Something like: "Detected variable names likely created by GP functions. Note that the cloned model will not work correctly with pre-existing GP objects. See issue: https://github.com/pymc-devs/pymc/issues/6883".

Reply:
Thanks @ricardoV94, I have made the suggestions and pushed the changes.